src/content/topics/integrations/Slack/managing.mdx

```diff
@@ -44,7 +44,31 @@ The table below explains the different filtering criteria:
     <TableHeadCell>Description</TableHeadCell>
   </TableHeader>
   <TableBody>
+     <TableRow>
+      <TableCell>Project / Environment</TableCell>
+      <TableCell>
+        Optional.
+        <br />
+        Filter by project, environment, or a combination of both.
+      </TableCell>
+    </TableRow>
     <TableRow>
+      <TableCell>Tag</TableCell>
```

Comment:
Ideally these would be in the same order as they're displayed in the app: https://monosnap.com/direct/fFGOcJpyjOl417Gg1ztmMVvTyfCjCF (meaning, move "Tag" down to the bottom). Also rename "Tag" to "Flag tag" and update the description; we don't support other kinds of tags such as segment tags.